firestore/docs/index.rst

```diff
@@ -1,5 +1,12 @@
 .. include:: README.rst
 
+.. note::
+
+   Because the firestore client uses :mod:`grpc` library and the third-party
```

Comment:
Firestore doesn't use `requests` at all.  We should refer to the `gRPC` library using its PyPI name, e.g., `grpcio`.